ci/dump-logs.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Dump the emulator log file. Tests run in the bigtable/tests directory.
+echo "================================================================"
+cat bigtable/tests/emulator.log >&2
```

Comment:
The macOS build and Bazel builds do not run the integration tests, presumably failures in `after_success` and `after_failure` are not a problem?

Reply:
done - I took a look at one of the runs in the PR, and the rest of the after_success script was not executed (i.e. the second echo ==== horizontal line), but the build passed.